corehq/sql_db/management/commands/configure_pl_proxy_cluster.py

```diff
@@ -137,21 +137,24 @@ def _get_alter_server_sql(cluster_name, shards_to_update):
 def create_pl_proxy_cluster(cluster_config, verbose=False, drop_existing=False):
     proxy_db = cluster_config.proxy_db
 
-    if drop_existing:
-        with connections[proxy_db].cursor() as cursor:
-            cursor.execute(get_drop_server_sql(cluster_config.cluster_name))
-
-    config_sql = get_pl_proxy_server_config_sql(cluster_config.cluster_name, cluster_config.get_shards())
-    user_mapping_sql = get_user_mapping_sql(cluster_config)
+    sql = get_sql_to_create_pl_proxy_cluster(cluster_config, drop_existing)
 
     if verbose:
         print('Running SQL')
-        print(config_sql)
-        print(user_mapping_sql)
+        for command in sql:
+            print(f'\t{command}')
 
     with connections[proxy_db].cursor() as cursor:
-        cursor.execute(config_sql)
-        cursor.execute(user_mapping_sql)
+        for command in sql:
+            cursor.execute(command)
```

Comment:
is there a reason to keep the prints separate from the actual execution? printing just before the execution could be useful in knowing where the command is if it hangs

Reply:
odd, I thought I changed that. I've updated and added a transaction